File: core/src/com/inkostilation/pong/commands/response/ResponseMessageCommand.java
package com.inkostilation.pong.commands.response;

import com.inkostilation.pong.commands.AbstractResponseCommand;

public class ResponseMessageCommand extends AbstractResponseCommand {

    private String text;

    public ResponseMessageCommand(String text) {
        this.text = text;
    }

    @Override
    public void execute() {
        System.out.println(text);
    }
}

File: src/com/netnplay/commands/ResponseMessageCommand.kt
package com.netnplay.commands

/**
 * Simple response command designed for testing purpose, which send [text] string to the client
 * and prints it using standard output stream.
 */
class ResponseMessageCommand(private val text: String) : AbstractResponseCommand<Any>() {
    override fun execute(input: Any) {
        println(text)
    }

}